src/env.h

```diff
@@ -272,6 +275,9 @@ namespace node {
   V(pipe_constructor_template, v8::FunctionTemplate)                          \
   V(process_object, v8::Object)                                               \
   V(promise_reject_function, v8::Function)                                    \
+  V(promise_unhandled_rejection, v8::Function)                                \
```

Comment:
I'd call these `array_from_function` and `promise_unhandled_rejection_function`.